packages/gatsby-theme-api-docs/src/components/resource/method/scopes.tsx

```diff
@@ -0,0 +1,42 @@
+import SpacingsStack from '@commercetools-uikit/spacings-stack';
+import { Markdown } from '@commercetools-docs/ui-kit';
+import Title from './title';
+import { css } from '@emotion/react';
+
+const scopesGridStyle = css`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-gap: 0.5rem;
+`;
+
+type ScopesProps = {
+  scopes: string[];
+};
+
+const Scopes = (props: ScopesProps) => {
+  const hasMultiColumnScope = props.scopes.length >= 10;
```

Comment:
I'm happy with the change, however I think @nkuehn had mentioned not to use hard limits but rather use a CSS approach. Maybe he can clarify what he had in mind

Reply:
In sloppy words, I meant "if you want columns make it columns".   It's a bit like "if you want a grid use a grid and not a table".   It only superficially works but all the layout decisions about edge case and responsive behavior will be more aligned with your intent if you use the CSS feature that is representing the intent instead of trying to emulate the behavior with other means. 

https://developer.mozilla.org/en-US/docs/Web/CSS/columns